docs/src/interface.md

```diff
@@ -11,11 +11,109 @@ Depth = 2
+Informally speaking, the exponential map takes a vector (think of a direction and a length) at one point and returns another point,
+which lies towards this direction at distance of the specified length. The logarithmic map does the inverse, i..e. given two points, what is the vector that “points towards” the other point.
+
+```@autodocs
+Modules = [Manifolds, ManifoldsBase]
+Pages = ["exp_log_geo.jl"]
+Order = [:function]
+```
+
+### Retractions and inverse Retractions
+
+The exponential and logarithmic map might be too expensive to evaluate or not be available in a very stable numerical way. Retractions provide a possibly cheap, fast and stable alternative.
+
+The following figure compares the exponential map [`exp`](@ref)`(M, p, X)` on the [`Circle`](@ref)`(ℂ)` (or [`Sphere`](@ref)`(1)` embedded in $ℝ^2$ with one possible retraction, the one based on projections. Note especially that ``\mathrm{dist}(p,q)=\lVert X\rVert_p`` while this is not the case for ``q'``.
+
+![A comparson of the exponential map and a retraction on the Circle.](assets/images/retraction_illustration_600.png)
+
+```@autodocs
+Modules = [Manifolds, ManifoldsBase]
+Pages = ["retractions.jl"]
+Order = [:function]
+```
+
+To distinguish different types of retractions, the last argument of the (inverse) retraction
+specifies a type. The following ones are available.
+
+```@autodocs
+Modules = [Manifolds, ManifoldsBase]
+Pages = ["retractions.jl"]
+Order = [:type]
+```
+
+### Projections
+
+A manifold might be embedded in some space.
+Often this is implicitly assumed, for example the complex [`Circle`](@ref) is embedded in the complex plane.
+Let‘s keep the circle in mind in the following as a simple example.
+For the general case see of explicitly stating an embedding and/or distinguising several, different embeddings, see [Embedded Manifolds](@ref EmbeddedmanifoldSec) below.
+
+To make this a little more concrete, let‘s assume we have a manifold ``\mathcal M`` which is embedded in some manifold ``\mathcal N`` and the image ``i(\mathcal M)`` of the embedding function ``i`` is a closed set (with respect to the topology on ``\mathcal N``). Then we can do two kinds of projections.
+
+To make this concrete in an example for the Circle ``\mathcal M=\mathcal C := \{ p ∈ ℂ | |p| = 1\}``
+the embedding can be chosen to be the manifold ``N = ℂ`` and due to our representation of ``\mathcal C`` as complex numbers already, we have ``i(p) = p`` the identity as the embedding function.
+
+1. Given a point ``p∈\mathcal N`` we can look for the closest point on the manifold ``\mathcal M`` formally as
+
+```math
+  \operatorname*{arg\,min}_{q\in \mathcal M} d_{\mathcal N}(i(q),p)
+```
+
+And this resulting ``q`` we call the projection of ``p`` onto the manifold ``\mathcal M``.
+
+2. Given a point ``p∈\mathcal M`` and a vector in ``X\inT_{i(p)}\mathcal N`` in the embedding we can similarly look for the closest point to ``Y∈ T_p\mathcal M`` using the push forward ``i_*`` of the embedding.
```

Comment:
```suggestion
2. Given a point ``p∈\mathcal M`` and a vector in ``X\inT_{i(p)}\mathcal N`` in the embedding we can similarly look for the closest point to ``Y∈ T_p\mathcal M`` using the pushforward ``i_*`` of the embedding.
```
We currently have a different notation for pushforward: https://juliamanifolds.github.io/Manifolds.jl/stable/misc/notation.html .

Reply:
Oh, indeed, I will change those, I somehow missed the notation difference (and recently used this one).